targets/TARGET_STM/qspi_api.c

```diff
@@ -35,7 +25,15 @@
 #include "pinmap.h"
 #include "PeripheralPins.h"
 
-// activate / de-activate debug
+#include "mbed_trace.h"
+
+#if defined(OCTOSPI1)
+#define TRACE_GROUP "ST_OSPI"
```

Comment:
"The group name length is limited to four characters." - https://github.com/ARMmbed/mbed-trace#compromises

Reply:
Oh...
I will correct.

PS: I suggest you a "git grep TRACE_GROUP".... :-(